src/main/java/com/shaft/support/DBActions.java

```diff
@@ -0,0 +1,155 @@
+package com.shaft.support;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.testng.Assert;
+
+import com.shaft.io.ReportManager;
+
+public class DBActions {
+
+	Connection connection;
+	Statement statement;
+	ResultSet resultSet;
+	String dbType;
+	String dbServerIP;
+	String dbPort;
+	String dbName;
+	String username;
+	String password;
```

Comment:
it is a must that all the variables are private

Reply:
Done